OpenSourcePolitics/decidim
# Fix display condition answer option

#### :tophat: What? Why?
When in a Questionnaire you create a Question with a Display Condition, you select a question that have 2 choices (Matrix Single option or Multiple options) and set the condition "Equal to" the dropdown list of the Answer option is empty.

#### :pushpin: Related Issues
- Related to # https://github.com/decidim/decidim/issues/10374


#### TESTING

1. Create a Questionnaire Template
2. Create a first mandatory question of Matrix Type Single option with 2 answer choices (cf screenshot one)
3. Create a 2nd Question with a display condition.
4. For the condition question, select the 1st question and then for the condition select "Equal to"
5. Click on the Answer option dropdown list
6. See that the dropdown list contains the 2 answer options of the first question (cf screenshot two)

Tasks
[X ] Add specs

### :camera: Screenshots (optional)
ONE => 
<img width="1020" alt="one" src="https://github.com/user-attachments/assets/84e63af8-f2cd-4819-90f1-f8ff09fbef80">
TWO =>
<img width="1004" alt="two" src="https://github.com/user-attachments/assets/a158cc6d-14ed-4078-825a-ea0350bb2363">
